docs/core-concepts/message-structure.mdx

```diff
@@ -30,6 +31,30 @@ Each message must specify a `role` that identifies the sender. Valid role format
 - Files
 - Named results
 
+## Metadata
+
+MessageParts can include optional `metadata` to provide additional context or semantic information. ACP supports two types of metadata:
+
+### Citation Metadata
+
+Used to reference information sources and provide attribution for content. Citations can be rendered as inline icons or text range annotations.
```

Comment:
```suggestion
Used to reference information sources and provide attribution for content. Citations are defined as range annotations in the generated content..
```

To avoid coupling with "UI".